packages/terra-status/tests/jest/Status.test.jsx

```diff
@@ -34,15 +34,6 @@ it('should render a arrange with status and customProps', () => {
   expect(wrapper).toMatchSnapshot();
 });
 
-// Prop Tests
-it('should have all props including customProps set correctly', () => {
```

Comment:
This test is covered by the test above. I would have switched this to a snapshot, but the tests on line 34 already does that for us.